projects/plugins/crm/src/automation/commons/actions/email/class-send-generic-email.php

```diff
@@ -0,0 +1,121 @@
+<?php
+/**
+ * Jetpack CRM Automation Send_Generic_Email action.
+ *
+ * @package automattic/jetpack-crm
+ * @since $$next-version$$
+ */
+
+namespace Automattic\Jetpack\CRM\Automation\Actions;
+
+use Automattic\Jetpack\CRM\Automation\Data_Types\Contact_Data;
+use Automattic\Jetpack\CRM\Entities\Contact;
+use Automattic\Jetpack\CRM\Entities\Factories\Contact_Factory;
+
+/**
+ * Adds the Send_Generic_Email class.
+ *
+ * @since $$next-version$$
+ */
+class Send_Generic_Email extends Base_Send_Email {
+
+	/**
+	 * Get the slug name of the step.
+	 *
+	 * @since $$next-version$$
+	 *
+	 * @return string The slug name of the step.
+	 */
+	public static function get_slug(): string {
+		return 'jpcrm/send_generic_email';
+	}
+
+	/**
+	 * Get the title of the step.
+	 *
+	 * @since $$next-version$$
+	 *
+	 * @return string The title of the step.
+	 */
+	public static function get_title(): string {
+		return __( 'Send email', 'zero-bs-crm' );
+	}
+
+	/**
+	 * Get the description of the step.
+	 *
+	 * @since $$next-version$$
+	 *
+	 * @return string The description of the step.
+	 */
+	public static function get_description(): string {
+		return __( 'Sends an email', 'zero-bs-crm' );
+	}
+
+	/**
+	 * Get the category of the step.
+	 *
+	 * @since $$next-version$$
+	 *
+	 * @return string The category of the step.
+	 */
+	public static function get_category(): string {
+		return __( 'General', 'zero-bs-crm' );
+	}
+
+	/**
+	 * Send a generic email.
+	 *
+	 * @since $$next-version$$
+	 *
+	 * @param mixed  $data Data passed from the trigger.
+	 * @param ?mixed $previous_data (Optional) The data before being changed.
+	 */
+	public function execute( $data, $previous_data = null ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
+
+		/** @var Contact $contact */
+		$contact = $data->get_data();
```

Comment:
The difference between the original PR and this is that the new one is now only sending to contacts, whereas mine was extensible (theoretically) to allow sending to any email, which is a far more useful action.

If you want to limit it, we at least should rename the action.

Reply:
I'll change the Action name by Send_Contact_Email